sunpy/net/dataretriever/sources/__init__.py

```diff
@@ -0,0 +1,2 @@
+# from . import adapt
```

Comment:
I assume we have made due without this file? Can you delete this nad we can add it back if we need it?